### ISSUE TITLE
Evaluation metrics

### ISSUE THREAD

**Issue by mortonjt** [mortonjt/mds-approximations#2]
Fortunately there are quite a few evaluation metrics already put into place.

We'll probably want to also add the following

- [ ] RMSE
- [ ] Frobenius norm
- [ ] [Aitchison stress](https://www.researchgate.net/publication/226223129_Dealing_with_Zeros_and_Missing_Values_in_Compositional_Data_Sets_Using_Nonparametric_Imputation)  see Equation 9

**Comment by cameronmartino** [comment]
Agreed, also adding Omega norm which will measure the convergence of two matrices. 

- [x]  RMSE
- [x]  Frobenius norm
- [x]  Omega norm
- [x] Aitchison stress see Equation 9
